Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -94,6 +104,12 @@ function vtkOpenGLImageMapper(publicAPI, model) {
     shaders.Geometry = '';
   };
 
+  publicAPI.replaceShaderCoincidentOffset = replaceShaderCoincidentOffset.bind(
+    null,
+    publicAPI,
+    model
+  );
```

Comment:
@jourdain is there something more elegant doable here ? would `bind(this)` work ?

Reply:
Just do like everything else (closure). 

`createReplaceShaderCoincidentOffset(publicAPI, model) => (...) => {}`

Reply:
notice the `create` which try to make things more explicit.